### PR TITLE
Makes minor updates to Network and Bonding options

### PR DESCRIPTION
**Pull request by stevsmit**
Makes minor updates to Network and Bonding options

Version(s):
4.8+

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2052635

Link to docs preview: https://deploy-preview-44946--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-bare-metal-network-customizations.html#installation-user-infra-machines-routing-bonding_installing-bare-metal-network-customizations

Requires SME/QE ack.

### CODE REVIEW

**Comment by djslavens** [comment]
verified.

File: modules/installation-network-user-infra.adoc [hunk 1]
```diff
@@ -148,7 +148,7 @@ ifndef::ibm-z,azure[]
 [id="installation-host-names-dhcp-user-infra_{context}"]
 == Setting the cluster node hostnames through DHCP
 
-On {op-system-first} machines, the hostname is set through NetworkManager. By default, the machines obtain their hostname through DHCP. If the hostname is not provided by DHCP, it is obtained through a reverse DNS lookup. Reverse DNS lookup occurs after the network has been initialized on a node and can take time to resolve. Other system services can start prior to this and detect the hostname as `localhost` or similar. You can avoid this by using DHCP to provide the hostname for each cluster node.
+On {op-system-first} machines, the hostname is set through NetworkManager. By default, the machines obtain their hostname through DHCP. If the hostname is not provided by DHCP or set statically through kernel arguments or another method, it is obtained through a reverse DNS lookup. Reverse DNS lookup occurs after the network has been initialized on a node and can take time to resolve. Other system services can start prior to this and detect the hostname as `localhost` or similar. You can avoid this by using DHCP to provide the hostname for each cluster node.
```
Comment (opayne1) [inline modules/installation-network-user-infra.adoc:151]: Would this make more sense to have it in like a listed format? Not sure if that does anything to the meaning.
Basically removing one "or" and adding a comma:
> 
If the hostname is not provided by DHCP, set statically through kernel arguments, or another method,

File: modules/installation-user-infra-machines-static-network.adoc [hunk 2]
```diff
@@ -100,6 +100,14 @@ ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none
 nameserver=4.4.4.41
 ----
 
+a|You can configure an IP address without assigning a static hostname. If a static hostname is not set by the user, it will be picked up and automatically set by a reverse DNS lookup. Use the same example sets as those above, and exclude the hostname.
```
Comment (opayne1) [inline modules/installation-user-infra-machines-static-network.adoc:103]: Just a [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#cloud-services-visual-info) note:
Last bullet about directional "up, down, left, right, etc." 

Could you say a variation of: 

> Use the same example sets as those mentioned in section a

**Review by opayne1 — COMMENTED** [review]
Just some tiny things I noticed. :)

File: modules/installation-network-user-infra.adoc (hunk 1 above)
Comment (stevsmit) [reply modules/installation-network-user-infra.adoc:151]: This whole sentence just read weird to me. I don't like how vague "another method" is. But I made your change and I think it helped!

Comment (opayne1) [reply modules/installation-network-user-infra.adoc]: It read weird to me too. I think your change looks great!

File: modules/installation-user-infra-machines-static-network.adoc (hunk 2 above)
Comment (opayne1) [reply modules/installation-user-infra-machines-static-network.adoc]: i like how you fixed! Makes more sense where it is now! :)

**Comment by opayne1** [comment]
LGTM!!! :)

**Pull request merged**